### PR TITLE
docs: optimize expression in routeLoader$()

### PR DESCRIPTION
**Pull request by JerryWu1234**
# What is it?

- Feature / enhancement
- Bug
- Docs / tests / types / typos
- Infra

#6929 

# Description

# Checklist

- [ ] My code follows the [developer guidelines of this project](https://github.com/QwikDev/qwik/blob/main/CONTRIBUTING.md)
- [ ] I performed a self-review of my own code
- [ ] I added a changeset with `pnpm change`
- [ ] I made corresponding changes to the Qwik docs
- [ ] I added new tests to cover the fix / functionality

### CODE REVIEW

**Review by gioboa — CHANGES_REQUESTED** [review]
Thanks @JerryWu1234 fro your help
is `**.**` a typo in your description?

**Comment by JerryWu1234** [comment]
> Thanks @JerryWu1234 fro your help is `**.**` a typo in your description?

done

File: packages/docs/src/routes/docs/(qwikcity)/route-loader/index.mdx [hunk 1]
```diff
@@ -22,7 +22,7 @@ created_at: '2023-03-20T23:45:13Z'
 
 Route Loaders load data in the server so it becomes available to use inside Qwik Components. They trigger when SPA/MPA navigation happens so they can be invoked by Qwik Components during rendering.
 
-Route Loaders can only be declared inside the `src/routes` folder, in a `layout.tsx` or `index.tsx` file, and they MUST be exported.
+Please note that route loaders should be exported only from `layout.tsx` or `index.tsx` files. But they can be declared in any valid way ES modules allow. To reuse a route loader across multiple `layout.tsx` or `index.tsx files, define it in a separate file, export it, then import it in `layout.tsx` or `index.tsx files and re-export it as a named export.
```
[the remark this answers is not in the document]
Comment (JerryWu1234) [reply packages/docs/src/routes/docs/(qwikcity)/route-loader/index.mdx:25]: I'll try it

**Review by gioboa — APPROVED** [review]
Thanks @JerryWu1234

**Pull request merged**